src/wp-includes/interactivity-api/class-wp-interactivity-api.php

```diff
@@ -272,6 +272,7 @@ public function process_directives( string $html ): string {
 	 * it returns null if the HTML contains unbalanced tags.
 	 *
 	 * @since 6.5.0
+	 * @since 6.6.0 The function displays a warning message when the HTML contains unbalanced tags.
```

Comment:
```suggestion
	 * @since 6.6.0 The function displays a warning message when the HTML contains unbalanced tags or a directive appears in a MATH or SVG tag.
```

Reply:
Done in https://github.com/WordPress/wordpress-develop/pull/6413/commits/3e64ebb6a8e94a13f937d221e110633167d61af0